### ISSUE TITLE
Do this tool support bigger file?

### ISSUE THREAD

**Issue by 837951602** [TheApeMachine/cloudtube#2]
I was doing such thing, encoding every 640*480-128 bits into one 2-color frame; but 1% of the bits when reading from frames are different to original 2-color images

**Comment by TheApeMachine** [comment]
Yeah, basically there is a problem with compression, which you can workaround. You would probably have to define color ranges to allow for some minor differences and also segment the frame into blocks of 4x4 (would be my guess) to act as a single "pixel".

**Comment by 837951602** [comment]
Then how much extra space would it likely cost as video?
